CHANGELOG.md

```diff
@@ -3,6 +3,59 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.15.0
+
+_Jul 28, 2022_
+
+We'd like to offer a big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Implement row pinning (#4863) @cherniavskii
+
+  Pro users can now [pin rows](https://mui.com/x/react-data-grid/rows/#row-pinning) on top or bottom of the grid.
+  To do so, enable the feature with `experimentalFeatures={{ rowPinning: true }}` and provide the row to pin with the `pinnedRows` prop.
+  For more details see the [documentation](https://mui.com/x/react-data-grid/rows/#row-pinning).
+
+- 🎁 Make aggregation public (#5592) @cherniavskii
+
+  Premium users can now add aggregated data in the grid.
+  Your user will be able to compute the average, sum, min, and other data with a single click.
+
+  ![modify aggregation](https://user-images.githubusercontent.com/45398769/181478185-142f87b0-c8de-4585-b381-cf4fdc62d99f.gif)
```

Comment:
```suggestion
  https://user-images.githubusercontent.com/13808724/181484439-1261f9ab-2c5c-4b70-9afd-0b0e20e053db.mp4
```
Should we try video instead of gif the way we did in aggregation blogpost?
https://deploy-preview-33595--material-ui.netlify.app/blog/aggregation-functions/

Reply:
Would they be auto-playable too?

Reply:
No, I don't think so. But I think it's still better than the gif

Reply:
Can we have a high res gif instead?

Reply:
Here's how it would look like in the changelog:
<img width="538" alt="Screenshot 2022-07-28 at 14 34 32" src="https://user-images.githubusercontent.com/13808724/181506122-190add70-3649-402b-b68d-3ff8c207edf9.png">

What do you think?

Reply:
True, @cherniavskii. It looks good. 
But I'm thinking again, and although the lack of auto-play is not bad, considering the length of the video (10 seconds), the lack of loop might potentially bring a bad experience. 

So, +1 to experiment with the video if it loops after play.

Reply:
> the lack of loop might potentially bring a bad experience.

Can you elaborate why? Personally, I feel like loop can be a bad user experience - especially in Gifs where it's not always clear whether is a long gif or it's just starting over and over 😀

Reply:
I think that for a user watching the feature in action for the first time (and without much context), 10 seconds will not be enough for them to understand it. 
So my bet is that they'd probably need to rewatch the loop a few times. 
In this case, if they need to press the play button every time, they will either be annoyed with the experience or give up on it.

Reply:
What do you think about using video then?
The quality is better, but no autoplay/loop.